Source/WebCore/animation/WebAnimation.h

```diff
@@ -255,6 +255,9 @@ class WebAnimation : public RefCounted<WebAnimation>, public EventTarget, public
     TimeToRunPendingTask m_timeToRunPendingPauseTask { TimeToRunPendingTask::NotScheduled };
     ReplaceState m_replaceState { ReplaceState::Active };
     uint64_t m_globalPosition { 0 };
+    RefPtr<CSSValue> m_specifiedRangeStart;
+    RefPtr<CSSValue> m_specifiedRangeEnd;
+
```

Comment:
I think these values should be higher up with other `RefPtr<>` types for padding purposes. There definitely should not be a new empty line here.

Reply:
Fixed.